Automattic/wp-calypso
# Use LanguagePicker in Calypso

#### Changes proposed in this Pull Request

* Use the newly extracted LanguagePicker from #46020 in the Calypso language picker modal

#### Testing instructions



* Verify that `/new?flags=gutenboarding/language-picker` (click the language picker in the upper right-hand corner), `/me/account` and `/settings/general/:site`'s language pickers match the spec from @olaolusoga and @ollierozdarz here: p9oQ9f-kT-p2
* Verify that search works as expected (it should filter exactly the same as the previous search implementation)
* Verify that selecting a new language and saving works
* Verify that selecting a new language and dismissing the modal without apply changes does not save the new language selection
